lib/Net/Twitter/Role/API/Media.pm

```diff
@@ -0,0 +1,103 @@
+package Net::Twitter::Role::API::Media;
+
+use Moose::Role;
+use Net::Twitter::API;
+use DateTime::Format::Strptime;
+use namespace::autoclean;
+
+
+has media_api_url => (
+    isa => 'Str',
+    is => 'rw',
```

Comment:
why rw vs ro?

Reply:
media_api_url potentially gets altered in the BUILD subroutine